homeassistant/components/bayesian/translations/en.json

```diff
@@ -1,8 +1,9 @@
 {
-  "issues": {
-    "manual_migration": {
-      "description": "The Bayesian integration now also updates the probability if the observed `to_state`, `above`, `below`, or `value_template` evaluates to `False` rather than only `True`. So it is no longer required to have duplicate, complementary entries for each binary state. Please remove the mirrored entry for `{entity}`.",
-      "title": "Manual YAML fix required for Bayesian"
+    "issues": {
```

Comment:
This should also be adjusted in strings.json, as that is the `root` for lokalize